pyproject.toml

```diff
@@ -47,11 +47,16 @@ dependencies = [
 ]
 
 [project.optional-dependencies]
-# note that dev dependencies are only pinned to major versions
-segmentation = ["cellpose"]
-sharding = ["tensorstore"]
+segment = ["cellpose"]
+
+track = ["ultrack @ git+https://github.com/royerlab/ultrack.git@v0.6.3"]
```

Comment:
Is this any different than the one from PyPI?

Reply:
Good question - I'm guessing no? @tayllatheodoro may know better